ironfish-rust/src/transaction/tests.rs

```diff
@@ -192,7 +192,7 @@ fn test_miners_fee() {
         NATIVE_ASSET_GENERATOR,
         spender_key.public_address(),
     );
```

Comment:
Is this worrying that this test was passing? I changed from receiver -> spender to get it to start passing.

Reply:
Don't think so - it seems like this test is just testing the miners fee is has the right amount and uses the hardcoded, technically invalid encryption keys. It is technically valid to pay out to any address prior to your change, but now it has some verification involved on the sender address which it didn't have before (since the sender address didn't exist)